File: src/main/java/macro303/neptunes/display/NeptunesApp.java
package macro303.neptunes.display;

import javafx.application.Application;
import javafx.scene.Scene;
import javafx.stage.Stage;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

/**
 * Created by Macro303 on 2018-05-08.
 */
public class NeptunesApp extends Application {

	public static void main(@Nullable String[] args) {
		launch(args);
	}

	@Override
	public void start(@NotNull Stage primaryStage) {
		var view = new NeptunesView();

		var scene = new Scene(view.getRoot());
		scene.getStylesheets().add("css/MaterialFX-v0.3.css");
		primaryStage.setTitle("Neptune's Pride");
		primaryStage.setScene(scene);
		primaryStage.setMaximized(true);
		primaryStage.show();
	}
}

File: src/main/kotlin/macro/neptunes/NeptunesApp.kt
package macro.neptunes

import javafx.stage.Stage
import tornadofx.*

/**
 * Created by Macro303 on 2018-05-08.
 */
class NeptunesApp : App(primaryView = NeptunesView::class, stylesheet = NeptunesStylesheet::class) {
	init {
		importStylesheet(stylesheet = "/css/modena_dark.css")
//		importStylesheet(stylesheet = "/css/bootstrap3.css")
//		importStylesheet(stylesheet = "/css/MaterialFX-v0.3.css")
//		importStylesheet(stylesheet = "/css/sdkfx.css")
		reloadStylesheetsOnFocus()
	}

	override fun start(stage: Stage) {
		super.start(stage)
		stage.isMaximized = true
	}
}